overlord/ifacestate/ifacestate_test.go

```diff
@@ -90,7 +92,11 @@ func (s *interfaceManagerSuite) SetUpTest(c *C) {
 	// TODO: transition this so that we don't load real backends and instead
 	// just load the test backend here and this is nicely integrated with
 	// extraBackends above.
-	s.restoreBackends = ifacestate.MockSecurityBackends([]interfaces.SecurityBackend{s.secBackend})
+	restoreBackends := ifacestate.MockSecurityBackends([]interfaces.SecurityBackend{s.secBackend})
+	s.restore = func() {
```

Comment:
I wonder if we should either use (en mass) `testutil.BaseTest` or remove it, we seem to have a swarm of _restore me later_ functions in all the suites.

Reply:
Good idea, done that.